src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/waypoints_pending_resumption_handler.cc

```diff
@@ -36,99 +36,104 @@ namespace sdl_rpc_plugin {
 
 SDL_CREATE_LOG_VARIABLE("SdlRPCPlugin")
 
+using hmi_apis::FunctionID::Navigation_SubscribeWayPoints;
+
+uint32_t get_corr_id_from_message(const smart_objects::SmartObject& message) {
```

Comment:
Use camel case

Reply:
Done in fface93be6b07b30b9fe605959ace2d9b18b8f77